packages/backend/src/services/ProjectService/ProjectService.ts

```diff
@@ -1,3 +1,4 @@
+import { NotFound } from '@aws-sdk/client-s3';
```

Comment:
I added the new streaming and save file functions into projectService, we should probably create a new `sqlRunnerService` or `streamingService` . If so, I'd like to do it on aseparate PR and move all relevant methods there too (eg: tables/fields)